modules/cnv-adding-virtio-drivers-vm-yaml.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-installing-virtio-drivers-on-existing-windows-vm.adoc
+// * cnv_users_guide/cnv-installing-virtio-drivers-on-new-windows-vm.adoc
+
+[id="cnv-adding-virtio-drivers-vm-yaml_{context}"]
+= Adding VirtIO drivers container disk to a virtual machine
+
+{ProductName} distributes VirtIO drivers for Microsoft Windows as a 
+container disk, available from the Red Hat Container Catalog. To install these 
+drivers to a Windows virtual machine, attach the `virtio-win` container disk to 
+the virtual machine as a SATA CD drive in the virtual machine configuration file. 
+
+[NOTE]
+====
+This procedure presumes the virtual machine is not running when you add the 
+container disk. If you add the container disk to a running virtual machine, you 
+can use `oc apply -f <vm.yaml>` in the CLI or reboot the virtual machine for the
+ changes to take effect.
+====
+
+.Procedure
+
+* Add the `cnv-tech-preview/virtio-win` container disk as a `cdrom` disk in the 
+Windows virtual machine configuration file. The container disk will be 
+downloaded from the registry if it is not already present in the cluster. 
+
+[source,yaml]
+----
+spec:
+  domain:
+    devices:
+      disks:
+        - name: virtiocontainerdisk
+          bootOrder: 2 <1>
+          cdrom:
+            bus: sata
+volumes:
+  - containerDisk:
+      image: cnv-tech-preview/virtio-win
+    name: virtiocontainerdisk
+----
+<1> {ProductName} boots virtual machine disks in the order defined in the 
+`VirtualMachine` configuration file. You can either define other disks for the 
+virtual machine before the `virtio-win` container disk, or use the optional 
+`bootOrder` parameter to ensure the virtual machine boots from the correct disk.
+ If you specify the `bootOrder` for a disk, it must be specified for all disks 
+in the configuration.
+
+After the virtual machine has started, the VirtIO drivers can be installed from 
```

Comment:
The VM can be started after the virtio-win is added to the VM specification. 
This would make more sense for me, if it were written as a step 2, telling me to start the VM and wait for the boot.

Reply:
As suggested, the admonition has been folded into a second step, which reads:
. The disk is available once the virtual machine has started:
** If you add the container disk to a running virtual machine, use
`oc apply -f <vm.yaml>` in the CLI or reboot the virtual machine for the changes
to take effect.
** If the virtual machine is not running, use `virtctl start <vm>`.